roles/webots/localisation.role

```diff
@@ -34,11 +34,22 @@ nuclear_role(
   # Localisation
   localisation::BallLocalisation
   localisation::FieldLocalisation
-  # Behaviour
-  behaviour::Controller
-  behaviour::skills::Getup
-  behaviour::skills::Stand
-  behaviour::skills::KickScript
-  behaviour::skills::DirectWalkController
-  behaviour::strategy::KeyboardWalk
+  # Director
+  extension::Director
+  # Servos
+  actuation::Kinematics
+  actuation::Servos
+  # Skills
+  skill::Walk
+  skill::FootController
+  skill::ScriptKick
+  skill::GetUp
+  skill::Look
+  # Planners
+  planning::GetUpPlanner
+  # planning::FallingRelaxPlanner Strategies
```

Comment:
!! we should properly get this working rather than remove it from all of the roles